game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -363,16 +362,12 @@ public boolean isShowing(final Territory territory) {
    * the units must all be in the same stack on the map, and exist in the given territory. call with
    * an null args
    */
-  void setUnitHighlight(final Map<Territory, List<Unit>> units) {
-    highlightedUnits = units;
+  public void setUnitHighlight(final Collection<List<Unit>> units) {
+    highlightedUnits = units == null ? Collections.emptyList() : units;
     SwingUtilities.invokeLater(this::repaint);
   }
 
-  Map<Territory, List<Unit>> getHighlightedUnits() {
-    return highlightedUnits;
-  }
-
-  public void centerOn(final @Nullable Territory territory) {
+  public void centerOnTerritoryIgnoringMapLock(final @Nullable Territory territory) {
     if (territory == null || uiContext.getLockMap()) {
```

Comment:
Notice there is still a `centerOn` which takes into account map lock and calls the `centerOnTerritoryIgnoringMapLock`:
```
  public void centerOn(final @Nullable Territory territory) {
    if (uiContext.getLockMap()) {
      return;
    }
    centerOnTerritoryIgnoringMapLock(territory);
  }
```

Reply:
From my understanding `centerOn` doesn't ignore the lock because it checks `uiContext.getLockMap()` and if this method returns true it returns. Lock checked.
But at the same time `centerOnTerritoryIgnoringMapLock` does the same check in the first line, so why have a second method if they both do the exact same?

Reply:
Ah! Funny what happens when you read what you actually have vs think you have. The 'lock' check in "ignoring" is a copy/paste bug, fixed in: e2d32d684

While there, the territory null check is moved to 'centerOn'.